app/views/users/edit.html.erb

```diff
@@ -0,0 +1,85 @@
+<section class="hero is-primary is-fullheight">
+  <div class="hero-head">
+    <nav class="navbar navbar">
+      <div class="navbar-brand">
+        <a class="navbar-item" href="/">
+          <span class="icon title">
+            <i class="fas fa-pastafarianism"></i>
+          </span>
+          <span class="title">mogumogu</span>
+        </a>
+        <span class="navbar-burger burger" data-target="navbarMenu">
+          <span>
+          </span>
+          <span>
+          </span>
+          <span>
+          </span>
```

Comment:
Is this needed? 👀

Reply:
Yes. Bulma needs this structure for the [navbar-burger component](https://bulma.io/documentation/components/navbar/#navbar-burger).